### ISSUE TITLE
Gutter error markers don't update upon entering playmode

### ISSUE THREAD

**Issue by Wipeless** [ImpowerGames/impower#168]
**Describe the bug**
When using the `>` command to go to a new section, if you complete the Section name starting with the autocomplete and then finishing the suggestion with the "return" key you'll see no errors in Edit Mode.  However, when you start the script in Play Mode, an error will generate.

**Steps To Reproduce**
1. Create a script using the go-to `>` command:
```
# Root

Time to try

> {~*|First|Second|Third}

^

# First

First try

> Root

# Second

Second try

> Root

# Third

Third try

> Root
```
2. On any of the go-to `>` commands, completely erase it.
3. Type `>` to prepare creating a destination for the go-to command.
4. As you begin to type "Root", stop at "Ro".
5. Use the auto complete suggestion for "Root" and push return to complete the section name.
6. Verify that you don't see any errors in Edit Mode.
7. Start the script.

**Expected Result**
You continue to not have errors in play mode since non existed in edit mode.

**Actual Result**
An error is generated at the line of your autocompleted go-to command.  It is angry since it cannot find the unknown section called "Ro" even when it's written as "Root".

**Screenshots**
![autoCompleteError](https://user-images.githubusercontent.com/1523864/161199095-bff98874-fba0-4866-8a44-5e5c6d6b20c1.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 99.0.4844.84

**Additional context**
This seems to work fine when you modify an existing go-to command Section.

For example, if you already have the existing line:
> Root

and then you removed the "ot" and then used autocomplete to finish the Section name this will work fine.

However, if you completely start a new section from scratch and then perform the autocomplete is when you'll see the error.

**Comment by lovelle-cardoso** [comment]
@Wipeless Fixed by 62213f6b

**Comment by Wipeless** [comment]
Verified!
